blog/2024-06-02-kqir-query-engine/index.md

```diff
@@ -0,0 +1,222 @@
+---
+slug: kqir-query-engine
+title: "KQIR: a query engine for Apache Kvrocks that supports both SQL and RediSearch queries"
+authors: [twice]
+---
+
+## Intro
+
+TL;DR:
+
+![ftsearch-demo](./images/ftsearch-demo.png)
```

Comment:
How do you prepare the demo data? I suppose a full (self-contain) example helps and can be explored more smoothly.

As a reader, I don't know the real data stored and thus can hardly understand these commands but only new syntax supported.

Reply:
The full demo is in the "Try it!" section (i.e. the last section).

Reply:
Then we can add a link to guide users there if one would like to see the full demo.

Reply:
Good idea!

Reply:
done